src/Couchbase.Lite.Shared/API/Database/Database.cs

```diff
@@ -731,12 +746,13 @@ public Document GetDocument([@NotNull]string id)
         }
 
         /// <summary>
-        /// Gets a list of index names that are present in the database
+        /// [DEPRECATED] Gets a list of index names that are present in the database
         /// </summary>
         /// <returns>The list of created index names</returns>
+        [Obsolete("GetIndexes is deprecated, please use GetDefaultCollection().GetIndexes")]
         [@NotNull]
         [@ItemNotNull]
-        public IList<string> GetIndexes()
+        public IReadOnlyList<string> GetIndexes()
```

Comment:
I don't think you can change this, it would be considered breaking from the existing API right?